docs/data/joy/components/menu/MenuIconSideNavExample.js

```diff
@@ -9,8 +10,6 @@ import Apps from '@mui/icons-material/Apps';
 import Settings from '@mui/icons-material/Settings';
 import Person from '@mui/icons-material/Person';
 
-// The Menu is built on top of Popper v2, so it accepts `modifiers` prop that will be passed to the Popper.
```

Comment:
Shouldn't be deleted. You can add these comments to the matching TS demo and run `yarn docs:typescript:formatted`.